crates/fluvio-benchmark/src/producer_worker.rs

```diff
@@ -60,18 +86,15 @@ impl ProducerWorker {
         println!("producer is sending batch");
 
         for record in self.records_to_send.into_iter() {
-            self.stat.start();
-            let time = std::time::Instant::now();
-            let send_out = self
+            self.stat.start().await;
```

Comment:
should remove this

Reply:
Start is setting the first instant to calculate the final bandwidth at the end.

Reply:
removed!